NitroxClient/Debuggers/SceneDebugger.cs

```diff
@@ -216,5 +312,238 @@ private void RenderTabScenes()
+            selectedObject.transform.localScale = selectedObjectScale;
+            if (sendToServer)
+            {
+                DebuggerAction.SendValueChangeToServer(selectedObject.transform, "enabled", selectedObjectActiveSelf);
+                DebuggerAction.SendValueChangeToServer(selectedObject.transform, "position", selectedObjectPos);
+                DebuggerAction.SendValueChangeToServer(selectedObject.transform, "rotation", selectedObjectRot);
+                DebuggerAction.SendValueChangeToServer(selectedObject.transform, "scale", selectedObjectScale);
+            }
+
+            foreach (DebuggerAction action in actionList)
+            {
+                action.SaveFieldValue();
+                if (sendToServer)
+                {
+                    action.SendValueChangeToServer();
+                }
+            }
+            actionList = new List<DebuggerAction>();
+        }
+    }
+
+    public class DebuggerAction
+    {
+        public Component Component;
+        public FieldInfo Field;
+        public object Obj;
+        public object Value;
+
+        public DebuggerAction(Component component, FieldInfo field, object obj, object value)
+        {
+            Component = component;
+            Field = field;
+            Obj = obj;
+            Value = value;
+        }
+
+
+        public void SaveFieldValue()
+        {
+            Field.SetValue(Obj, Value);
+        }
+
+        public void SendValueChangeToServer()
+        {
+            if (Multiplayer.Main != null)
+            {
+                string guid = GetGameObjectPath(Component.gameObject);
+                int id = GetGameObjectChildNumber(Component.gameObject);
+                int id2 = GetComponentChildNumber(Component);
+                Multiplayer.Logic.Debugger.SceneDebuggerChange(guid, id, id2, Field.Name, Value);
+            }
+        }
+
+        public static void SendValueChangeToServer(Component component, string fieldName, object value)
+        {
+            if (Multiplayer.Main != null)
+            {
+                string path = GetGameObjectPath(component.gameObject);
+                int objectNumber = GetGameObjectChildNumber(component.gameObject);
+                int componentNumber = GetComponentChildNumber(component);
+                Multiplayer.Logic.Debugger.SceneDebuggerChange(path, objectNumber, componentNumber, fieldName, value);
+            }
+        }
+
+        private static string GetGameObjectPath(GameObject gameObject)
+        {
+            string path = gameObject.name;
+            Transform parent = gameObject.transform;
+            while (parent.parent != null)
+            {
+                parent = parent.parent;
+                path = parent.name + "/" + path;
+            }
+            return "/" + path;
+        }
+
+        private static int GetGameObjectChildNumber(GameObject gameObject)
+        {
+            int childNumber = -1;
```

Comment:
You can replace this method with `return gameObject.transform.GetSiblingIndex()`: https://docs.unity3d.com/ScriptReference/Transform.GetSiblingIndex.html